### ISSUE TITLE
sed stuff does not work on linux

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-umbrella#52]
```
[dietrich@thinkpad-x250 xtext-umbrella]$ sed -i '' "s?'http://services.typefox.io/open-source/jenkins/job/xtext-web/job/[^/]*/lastSuccessfulBuild/artifact/build/maven-repository/'?jenkinsPipelineRepo('xtext-web')?" /home/dietrich/xtext-master/git/xtext-xtend/gradle/upstream-repositories.gradle
sed: s?'http://services.typefox.io/open-source/jenkins/job/xtext-web/job/[^/]*/lastSuccessfulBuild/artifact/build/maven-repository/'?jenkinsPipelineRepo('xtext-web')? kann nicht gelesen werden: Datei oder Verzeichnis nicht gefunden

```

**Comment by cdietrich** [comment]
i wonder if sth like `sed -i.bak` instead of `sed -i ''` will work on mac

**Comment by cdietrich** [comment]
alternatively do a os detection and specify `-i` for non mac and `-i ''` for mac

**Comment by cdietrich** [comment]
proposal:

```
if [[ "$OSTYPE" == "darwin"* ]]; then
	sed_inplace="sed -i ''"
else
	sed_inplace="sed -i"
fi
```

TODO: test with a mac

**Comment by cdietrich** [comment]
Fixed in 2.14
